.travis.yml

```diff
@@ -36,9 +36,9 @@ script:
   # this checks that the source code follows the Symfony Code Syntax rules
   - '[[ "$TRAVIS_PHP_VERSION" == "7.4snapshot" ]] || ./vendor/bin/php-cs-fixer fix --diff --dry-run -v'
   # this checks that the YAML config files contain no syntax errors
-  - ./bin/console lint:yaml config
+  - ./bin/console lint:yaml config --parse-tags
```

Comment:
Probably lint Yaml in with `--env=prod` as well?

Reply:
there is no difference between `dev` or `prod` in this case